inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -414,43 +414,31 @@ CNNNetworkNGraphImpl::reshape(const std::map<std::string, std::vector<size_t>>&
     }
 }
 
-StatusCode CNNNetworkNGraphImpl::serialize(const std::string& xmlPath, const std::string& binPath,
+StatusCode CNNNetworkNGraphImpl::serialize(const std::string& xmlPath,
+                                           const std::string& binPath,
                                            ResponseDesc* resp) const noexcept {
-    auto network = cnnNetwork;
-    if (!network) {
-        // TODO: once Serialization::SerializeV10 supports true IR v10
-        // remove this conversion and WA for execution graph
-        try {
-            bool isExecutionGraph = true;
-            for (const auto & op : _ngraph_function->get_ops()) {
-                auto & rtInfo = op->get_rt_info();
-                if (rtInfo.find(ExecGraphInfoSerialization::PERF_COUNTER) == rtInfo.end()) {
-                    isExecutionGraph = false;
-                    break;
-                }
-            }
-            if (isExecutionGraph) {
-                Serialization::SerializeV10(xmlPath, binPath, (InferenceEngine::ICNNNetwork&)*this);
-                return OK;
-            }
-
+    try {
+        if (getFunction()) {
+            ngraph::pass::Manager manager;
+            manager.register_pass<ngraph::pass::Serialize>(xmlPath, binPath);
+            manager.run_passes(_ngraph_function);
+        } else {
 #ifdef ENABLE_V7_SERIALIZE
```

Comment:
we should remove `ENABLE_V7_SERIALIZE` and IR v7 path since we have ngraph one.

Reply:
I was told by @ilyachur that it will be removed in separate PR.

Reply:
@ilya-lavrenov We discussed it, and my proposal is to don't combine these activities and remove v7 serialiser in another PR (because removing of v7 serialiser affects several files).

Reply:
@ilya-lavrenov is that ok for you? Can I resolve this conversation?

Reply:
sure.